dd-sdk-android/src/main/kotlin/com/datadog/android/log/internal/user/DatadogUserInfoProvider.kt

```diff
@@ -23,6 +23,16 @@ internal class DatadogUserInfoProvider(
         internalUserInfo = userInfo
     }
 
+    override fun setExtraProperties(properties: Map<String, Any?>) {
```

Comment:
Here I'd change the name to `addProperties` or `addUserProperties`; because we're adding the properties in parameter to the previous ones (overwriting some properties is a side effect).